privacy-policy/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: page
+title: Privacy Policy
+---
+
+# Vox Pupuli Privacy Policy
+
+This document details the data the Vox Pupuli collects about you, why it is
+collected, what is done with it, and your options for how it's processed.
+
+The Vox Pupuli project does not share data with 3rd parties, except for hosting
+requirements (e.g. GitHub, YouTube) as detailed below, or when required to by
+law.
+
+# Types of data we collect
+
+* Contact details
+  * Name
+  * E-Mail address
+
+# How and why we use your data
+
+## The website
+
+The Vox Pupuli website is a static site that does not track, collect, or store 
+infromation or set cookies. There is no account needed or required to access 
+the website itself. The website itself is hosted with [Github Pages](https://pages.github.com/) 
+and any data gathered by Github will adhere to the [Github Privacy Policy](https://help.github.com/articles/github-privacy-statement/).
+
+The website does host a community run Blog series and do require that the 
+author's name be visible in the blog post indefinitely.
```

Comment:
We often add an author, but it is not required: https://voxpupuli.org/blog/2018/02/06/voxpupuli-community-journey/